addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -69,9 +69,6 @@ export default class PivotCorePlugin extends CorePlugin {
                     return CommandResult.InvalidNextId;
                 }
                 break;
-            case "REMOVE_GLOBAL_FILTER":
```

Comment:
why is this removed?

Reply:
because it's in the allowDispatch :D  it was a duplication of the function `_setPivotFieldMatching` that I introduced in the refactoring and I managed to find the existence of the anomally through one of the tests but can't really tell you which one :/  Most likely, the fix worked for every ds except pivots because their history was corrupted by this line